lib/unit_tests_utils/postgresql_web_service_client.rb

```diff
@@ -278,11 +284,20 @@ def cluster_status
     nodes.select { |node| node != nil }
   end
 
+  def manifest_name
+    manifest_name_opt.nil? ? manifest.name : manifest_name_opt
```

Comment:
Is this the same as?

```suggestion
    manifest_name_opt || manifest.name
```

Reply:
Done